driver/tensor_driver.hpp

```diff
@@ -144,4 +144,19 @@ size_t GetTensorSize(miopenTensorDescriptor_t& tensor)
 
     return sz;
 }
+
+size_t GetTensorSpace(miopenTensorDescriptor_t& tensor)
```

Comment:
Can you please add a comment that describes the difference between `GetTensorSize()` and `GetTensorSpace()`? In other words, when `GetTensorSpace()` must be used?

PS Re-runing CI job is not necessary except formatting check, so please can terminate the job after adding the comment.

Reply:
I think I should also add an assert about the incorrect use of 'GetTensorSize()'. That is extremely easy to step in, and it may not be immediately obvious due to how memory is allocated on GPU. With an 'assert', there would be at least some additional chance future development time would be saved due to immediate failure in debug on any incorrect use, rather than only with a large enough difference between requested and required memory.

Reply:
Very good, thanks!